src/iop/demosaic.c

```diff
@@ -56,7 +60,8 @@ typedef enum dt_iop_demosaic_method_t
   // methods for x-trans images
   DT_IOP_DEMOSAIC_VNG = DEMOSAIC_XTRANS | 0,
   DT_IOP_DEMOSAIC_MARKESTEIJN = DEMOSAIC_XTRANS | 1,
-  DT_IOP_DEMOSAIC_MARKESTEIJN_3 = DEMOSAIC_XTRANS | 2
+  DT_IOP_DEMOSAIC_MARKESTEIJN_3 = DEMOSAIC_XTRANS | 2,
+  DT_IOP_DEMOSAIC_FDC = DEMOSAIC_XTRANS | 4
```

Comment:
It seems that 3 is already reserved for "passthrough (monochrome) (experimental)", the fourth demosaicking option for xtrans.